charts/kube-prometheus-stack/values.yaml

```diff
@@ -875,6 +875,8 @@ kubeControllerManager:
     #   component: kube-controller-manager
 
   serviceMonitor:
+    enabled: true
```

Comment:
why would it be disabled ?

Reply:
For this use case you don't need the scrape job for controller-manager or scheduler because with k3s there is a single metrics endpoint for all three control plane services, and the apiserver scrape job/ServiceMonitor picks up the metrics for all of them.

Reply:
Why cannot it be configured in k3s instead of modifying a more generic and independent tool ?

Reply:
Should we add any condition/checks to this chart based on the opinionated decisions of every Kubernetes installer ? I feel it'd be a wrong decision.

Reply:
I definitely see where you're coming from. The chart is already quite complex, let's not increase that complexity unnecessarily.

I'll see if there is some other approach we can take either in configuration of the Helm chart as it stands or a PR against k3s. However I did attempt this before making this PR, and I really don't want to maintain a fork.